doc/source/user_guide/boundary_conditions.rst

```diff
@@ -61,25 +69,51 @@ boundary conditions at inlets.
 
 Copying boundary conditions
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
-The ``define.boundary_conditions.copy_bc`` TUI command copies boundary conditions
-to other zones.
+The following example shows a comparison between the TUI command and the
+python code for copying boundary conditions to other zones.
+
+TUI command
+
+.. code:: scheme
+
+    /define/boundary-conditions/copy-bc
+
+Python command
 
 .. code:: python
 
     session.solver.tui.define.boundary_conditions.copy_bc('cold-inlet','hot-inlet','()')
 
 Listing zones
 ~~~~~~~~~~~~~
-The ``define.boundary_conditions.list_zones`` TUI command prints in the Fluent console
-the types and IDs of all zones.
+The following example shows a comparison between the TUI command and the
+python code for printing the types and IDs of all zones in the Fluent
+console.
+
+TUI command
+
+.. code:: scheme
+
+    /define/boundary-conditions/list-zones
+
+Python command
 
 .. code:: python
 
     session.solver.tui.define.boundary_conditions.list_zones()
 
 Modifying cell zone conditions
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-The ``define.boundary_conditions.fluid`` TUI command modifies cell zone conditions.
+The following example shows a comparison between the TUI command and the
+python code for modifying cell zone conditions.
+
+TUI command
+
+.. code:: scheme
+
+    /define/boundary-conditions/fluid
```

Comment:
Does this take arguments? If so then it would be good to include those.

Reply:
@dnwillia-work Original purpose was just to mention a few high level TUI commands (from Fluent's documentation), but I agree to add arguments everywhere for the consistency as we are comparing the exact scheme code with the python code.